lib/drone/chef/config.rb

```diff
@@ -149,6 +139,17 @@ def keyfile_path
         )
       end
 
+      def logger
+        @logger ||= begin
+          l = Logger.new(STDIN)
```

Comment:
In such cases I end up with

Logger.new(STDOUT).tap do |l|
  like.level =...
end

Than you can skip and will never forget to return the instance